packages/core/src/PartialExecutionUtils/__tests__/DirectedGraph.test.ts

```diff
@@ -43,8 +43,8 @@ describe('DirectedGraph', () => {
 	});
 
 	//    ┌─────┐    ┌─────┐──► null
-	//    │node1├───►│node2|   ┌─────┐
-	//    └─────┘    └─────┘──►│node3|
+	//    │node1├───►│node2│   ┌─────┐
+	//    └─────┘    └─────┘──►│node3│
```

Comment:
Visual fix for the diagram. This is only visible in code editor.

Reply:
define "good font", so the test of us can also install it 😅

Reply:
Of course, I'm using JetBrainsMono Nerd Font and I got this:

![image](https://github.com/user-attachments/assets/25e843fa-96ae-4b7e-bf70-b50e134da2b1)

After changing it to the unicode character, I get this:

![image](https://github.com/user-attachments/assets/9d17b068-a94c-4b17-8415-74d5404d1416)